client/components/navigation/Navbar.tsx

```diff
@@ -46,69 +53,172 @@ function Navbar() {
           </HStack>
         </Link>
 
-        <HStack spacing={{ base: 2, sm: 7 }}>
-        <Link
-          color="black"
-          p="0.45rem"
-          m="0 0.25rem"
-          cursor="pointer"
-          textDecoration="none"
-          _hover={{textDecoration: "none", bgColor:"#e2e8f0"}}
-          _focus={{textDecoration: "none", border:"none"}}
-          borderRadius="18px"
-          href="/gigs">Gigs</Link>
-        <Link
-          color="black"
-          p="0.45rem"
-          m="0 0.25rem"
-          cursor="pointer"
-          textDecoration="none"
-          borderRadius="18px"
-          _hover={{textDecoration: "none", bgColor:"#e2e8f0"}}
-          _focus={{textDecoration: "none", border:"none"}}
-          href="/jobs">Jobs</Link>
+        <Button bg="gray.300" display={{ sm: 'block', md: 'none' }} zIndex={11}>
+          {isHamburgerOpen ? (
+            <CloseIcon
+              h={6}
+              w={6}
+              onClick={() => setIsHamburgerOpen(!isHamburgerOpen)}
+            />
+          ) : (
+            <HamburgerIcon
+              h={7}
+              w={7}
+              onClick={() => setIsHamburgerOpen(!isHamburgerOpen)}
+            />
+          )}
+        </Button>
+        {isHamburgerOpen && (
+          <VStack
+            position="fixed"
+            zIndex={10}
+            top={0}
+            left={0}
+            h="100%"
+            w="100%"
+            display={{ sm: 'flex', md: 'none' }}
+            bg="white"
+            spacing={10}
+            fontSize="3xl"
+            pt={20}
+          >
+            <Link
+              color="black"
+              p="0.45rem"
+              m="0 0.25rem"
+              cursor="pointer"
+              textDecoration="none"
+              borderRadius="18px"
```

Comment:
We shouldn't need to define any of these:

- `pointer` is the default browser cursor for links
- `textDecoration` is already `none` for Chakra links
- `borderRadius` doesn't do anything for us since they don't have a background color

Reply:
My mistake again! However, I was using the `borderRadius` for the background on `_hover`. This way it doesn't look squared but like a button as you hover/click on it.

Reply:
Ohhh that's a good point, hadn't thought of that! That makes perfect sense